obs2ioda-v2/README.md

```diff
@@ -1,11 +1,62 @@
 # obs2ioda-v2.x
-
-```
-Compile: cd obs2ioda-v2/src; make
-```
-NCEP BUFR library (https://github.com/NOAA-EMC/NCEPLIBS-bufr) is required to compile ``obs2ioda-v2.x``.  
-Edit obs2ioda-v2/src/Makefile to set proper BUFR_LIB before ``make``.
-
+## Installation
+obs2ioda-v2 utilizes CMake as its primary build system. Follow the steps below to build the project:
+
+### Prerequisites
+Please make sure the following libraries are installed:
+- NetCDF
+- NCEP BUFR library. (Instructions for installing the NCEP BUFR library are provided in a subsequent section)
+
+If you have an environment preconfigured for `mpas-jedi`, simply source that environment prior to building `obs2ioda`.
+
+### Build Instructions
+1. First, clone the repository into your preferred directory (`<OBS2IODA_ROOT_DIR>`):
+   ```bash
+   git clone https://github.com/NCAR/obs2ioda.git <OBS2IODA_ROOT_DIR>
+   ```
+2. Create a new directory `build` and navigate into it:
+   ```bash
+   mkdir build && cd build
+   ```
+3. Locate the NCEP BUFR library by executing the following command in the `NCEP BUFR` library's build directory:
+   ```bash
+   find . -name *libbufr*
+   ```
+4. Next, run CMake to configure the build. Remember to specify the path to the NCEP BUFR library:
+   ```bash
+   cmake <OBS2IODA_ROOT_DIR> -DNCEP_BUFR_LIB=<NCEP_BUFR_LIB_PATH>
```

Comment:
@amstokely , would you please update README.md to clarify `NCEP_BUFR_LIB_PATH` should point to libbufr.a ? such as, /glade/work/$user/bufr/build/src/libbufr.a   
Many thanks.

Reply:
Looks like NCEP_BUFR_LIB_PATH can be configured in the command line.